cli.go

```diff
@@ -19,8 +19,15 @@ func main() {
 	var installOpenAsar = flag.Bool("install-openasar", false, "Install OpenAsar on a Discord install")
 	var uninstallOpenAsar = flag.Bool("uninstall-openasar", false, "Uninstall OpenAsar from a Discord install")
 	var updateFlag = flag.Bool("update", false, "Update your local Vencord files")
+	var dir = flag.string("dir", false, " Select the location of your Discord client")
+	var client = flag.string("client", "default", "Select the branch of Discord you wish to modify [default|stable|ptb|canary]")
```

Comment:
have you tested this by building and running it? These method names for example should be uppercase: `flag.String` and Alex's feedback below also suggests that this will not work in the current state. You should use an IDE with support for go such as Goland or VSCode (https://code.visualstudio.com/docs/languages/go)